### PR TITLE
vmm: add config files for clear linux

### PR DESCRIPTION
**Pull request by GangSecurity**
configs for gordon peak

Signed-off-by: Chen Gang G <gang.g.chen@intel.com>

### CODE REVIEW

**Comment by syan10** [comment]
good to me

**Comment by syan10** [comment]
good to me

File: product/linux_trusty.cfg [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#For BXT-P on clear linux
+EVMM_CMPL_FLAGS :=
+#EVMM_CMPL_FLAGS += -DDEBUG
+
+EVMM_CMPL_FLAGS += -DLOG_LEVEL=3
```
Comment (fofeir) [inline product/linux_trusty.cfg]: are you sure log level 3 is required? normally we set it 0 for release.

Comment (GangSecurity) [reply product/linux_trusty.cfg]: change to 0, updated

**Pull request merged**